Formula/snowball.rb

```diff
@@ -0,0 +1,22 @@
+class Snowball < Formula
+  desc "Stemming algorithms"
+  homepage "https://snowballstem.org"
+  url "https://github.com/snowballstem/snowball/archive/refs/tags/v2.2.0.tar.gz"
+  sha256 "425cdb5fba13a01db59a1713780f0662e984204f402d3dae1525bda9e6d30f1a"
+  license "BSD-3-Clause"
+
+  def install
+    system "make"
+
+    lib.install "libstemmer.a"
```

Comment:
Seems much nicer to ship a shared library. Can we ask upstream if that's something they'd support?

Reply:
There's an [open issue](https://github.com/snowballstem/snowball/issues/34) and [PR](https://github.com/snowballstem/snowball/pull/160/files) for it (for Linux, but can update for Mac if accepted).